src/Faker/Provider/pt_BR/PhoneNumber.php

```diff
@@ -14,12 +14,21 @@ class PhoneNumber extends \Faker\Provider\PhoneNumber
     protected static $cellphoneFormats = array('9####-####');
 
     /**
-     * Generates a 2-digit area code not composed by zeroes.
+     * Pick one random entry out of an array of area codes.
      * @return string
```

Comment:
please add the link to http://www.anatel.gov.br/legislacao/resolucoes/16-2001/383-resolucao-263 in a phpDoc `@link` line.